typescript/submissionEvents.d.ts

```diff
@@ -28,6 +27,17 @@ export type CallbackSubmissionEvent = FormSubmissionEventConditional & {
   isDraft: boolean
 }
 
+type PdfSubmissionEventEmailTemplate =
+  | {
+      mustacheTag: string
+      type: 'FORM_ELEMENT'
```

Comment:
There are a bunch of others we need to support as well. See the AC.

Reply:
Wondering whether those should just be replaceable values inside the `text`
```
"Blah blah blah {EXTERNAL_ID} blah blah"
```